CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## \[Unreleased\]
+### Added
 - Add with_subset_dirs decorator
   (<https://github.com/openvinotoolkit/datumaro/pull/816>)
```

Comment:
Could you add the description for Datumaro binary format support here?

Reply:
I'll do in the next subsequent PR. There are many PRs piled up right now.

Reply:
- #828 will update `CHANGELOG.md`.